person.rb

```diff
@@ -7,6 +7,7 @@ def initialize(age, name = 'Unknown', parent_permission: true)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
   attr_accessor :name, :age
```

Comment:
- [ ]  A rental should be written/read in the book class. Kindly add rentals to the attr_accessor.